OpenRA.Mods.Common/AI/States/GroundStates.cs

```diff
@@ -20,6 +20,11 @@ protected virtual bool ShouldFlee(Squad owner)
 		{
 			return base.ShouldFlee(owner, enemies => !AttackOrFleeFuzzy.Default.CanAttack(owner.Units, enemies));
 		}
+
+		protected Actor FindClosestEnemy(Squad owner)
+		{
+			return owner.Bot.FindClosestEnemy(owner.Units.FirstOrDefault().CenterPosition);
```

Comment:
Do we need `FirstOrDefault` here? I guess it'll crash anyway if `FirstOrDefault` returns `null` (as we ask for the `CenterPosition` of `null`), so we can use just `First` instead?